src/plugins/intel_npu/src/plugin/npuw/llm_compiled_model.cpp

```diff
@@ -112,14 +102,16 @@ class TransposeValueTensors_llama3 : public TransposeValueTensors {
     void register_matcher_llama3(Context::Ref ctx) {
         auto param = opp::wrap_type<ov::op::v0::Parameter>();
         auto transpose = opp::wrap_type<ov::op::v1::Transpose>({opp::any_input(), opp::any_input()});
-        auto concat = opp::wrap_type<ov::op::v0::Concat>({param, transpose});
+        auto convert = opp::optional<ov::op::v0::Convert>({param->output(0)});
+        auto concat  = opp::wrap_type<ov::op::v0::Concat>({convert, transpose});
 
         // only difference is that broadcast wrapped into unsquese/reshape, while transposed tensor didn't change
         const auto unsqueeze_axes = opp::wrap_type<ov::op::v0::Constant>();
         auto unsqueeze = opp::wrap_type<ov::op::v0::Unsqueeze>({concat, unsqueeze_axes});
         auto broadcast = opp::wrap_type<ov::op::v1::Broadcast, ov::op::v3::Broadcast>({unsqueeze, opp::any_input()});
         auto reshape = opp::wrap_type<ov::op::v1::Reshape>({broadcast, opp::any_input()});
 
+        // v8 softmax? what? can be other softmaxes
```

Comment:
Theoretically there can be softmax with different opset version and patter will be broken

Reply:
created an issue: E-159463